### DISCUSSION TITLE
Why hepatic clearance could surpass hepatic flow rate in PK-Sim?

### DISCUSSION THREAD

**Discussion by jzq90** [Open-Systems-Pharmacology/Forum#756] [Questions & Problems]
Hi all,

I used a dummy enzyme which only expressed in the `liver` and adjusted the` in vitro specific clearance` to see what happens. However, in an IV simulation I found the `total body clearance` will increase with the increase of `in vitro  specific clearance` without upper limitation.

It is very weird that the total clearance is much higher than the hepatic blood flow rate (~21 ml/min/h). How to explain this？

![图片](https://user-images.githubusercontent.com/58142266/80275790-23ed3580-8716-11ea-9c4a-7d84c4d2d757.png)

![图片](https://user-images.githubusercontent.com/58142266/80275836-98c06f80-8716-11ea-910e-dc1bae819d12.png)

**Comment by prvmalik** [comment]
Hi there,

I get a similar result, though I do get an upper limit on plasma clearance.

Test compound:
MW 300g/mol, logP 1.00, fu 1.00, acid-base: neutral, KMET in vitro specific clearance

Test Individual:
European, male, age 30 years, default parameters (73kg), add KMET in liver periportal+pericentral

Liver blood flow rate is 0.43 L/min from the anatomy and physiology tab. We are interested in plasma flow (HCT = 0.47) so that gives us 0.20 L/min.

At very high specific clearances (e.g. > 5000 1/min), the PK clearance caps out at 16.28 mL/min/kg. For our individual, this is 1.18 L/min.

So plasma clearance is calculated as 6-fold higher than liver plasma flow...thinking on this.

Paul

**Comment by StephanSchaller** [comment]
> Liver blood flow rate is 0.43 L/min from the anatomy and physiology tab. We are interested in plasma flow (HCT = 0.47) so that gives us 0.20 L/min.

Two points you might be overlooking:
1. The liver blood flow depicted in PK-Sim is ARTERIAL blood flow only. You will have to add portal vein blood flow.
2. As an upper limit, we are considering blood flow (not plasma only) as, in theory, also compound in blood cells can diffuse into liver tissue in a single pass-through.


Maybe this helps.

Regarding the issue from @jzq90, it might be of interest how the simulated output (the plotted curve) looks like.

**Comment by prvmalik** [comment]
@StephanSchaller  Great point about the portal flow. Oops!

I've observed that plasma clearances can blow out of proportion at high lipophilicity (>= 3). Will repost if I discover an explanation.

**Comment by StephanSchaller** [comment]
> I've observed that plasma clearances can blow out of proportion at high lipophilicity (>= 3).

This may depend on the simulation time you are using. you might have distribution effects that add to apparent plasma clearance. After all, plasma clearance is calculated from the shape of the curve (if I am not mistaken).

So if you simulate for a very long time, this effect should be lower...

**Comment by jzq90** [comment]
@prvmalik @StephanSchaller Thanks you both. I get the reason based on your valuable ideas.
When `lipophilicity` increase, more drugs enter into the blood cells. The `Total body clearance` in the software is calculated by the concentration in the chamber displayed. 
The clearance calculated based on plasma could be very high when most of the drug enter blood cells, while the clearance calculated based on blood could not surpass the hepatic blood flow rate.
![图片](https://user-images.githubusercontent.com/58142266/80898747-9a52ee80-8d39-11ea-8879-da46194835ea.png)

**Comment by jzq90** [comment]
Another question is that the blood/plasma ratio increased too quickly with lipophilicity:
When lipophilicity=2.5, B/P is 2.5
When lipophilicity=3, B/P is 3
When lipophilicity=3.5, B/P is 18

This seems not reasonable because the B/P ratio of most drugs ranges from 0.5~2, however the situation when LogP exceeds 3 is not rare.
Maybe it is not a good idea to use LogP as a surrogate for lipophilicity in PK-sim.

**Comment by StephanSchaller** [comment]
> Maybe it is not a good idea to use LogP as a surrogate for lipophilicity in PK-sim.

This depends on the calculation method you use.
"PKSim standard" and "Schmitt" e.g. require LogMA. Others (R&R) require LogD or P:

LogD is the descriptor for lipophilicity of ionizable compounds as it accounts for the pH dependence of a molecule in aqueous solution. LogP describes lipophilicity for neutral compounds only, and should be applied with caution, especially when working with ionizable compounds.

**Comment by jzq90** [comment]
> This depends on the calculation method you use.
> "PKSim standard" and "Schmitt" e.g. require LogMA. Others (R&R) require LogD or P:
> 
Thanks for the explanation.
I've compared the blood/plasma ratio of a dummy neutral compound with Log P=3.5 having low intrinsic clearance in the liver (high intrinsic clearance show a similar trend).

  | PK-Sim standard | Rodgers and Rowland | Schemitt | Poulin and Theil | Berezhkovskiy
-- | -- | -- | -- | -- | --
Plasma   clearance (mL/min/kg) | 78.4 | 3.61 | 93.14 | 0.20 | 0.20
Blood   clearance  (mL/min/kg) | 4.26 | 0.2 | 3.66 | 0.01 | 0.01
Blood/Plasma | 18.4 | 18.1 | 25.4 | 20.0 | 20.0
Plasma   Vss (L/kg) | 236 | 106 | 125 | 31.6 | 31.6
Blood   Vss  (mL/min/kg) | 9.83 | 5.75 | 4.93 | 1.72 | 1.72
Thalf   (h) | 35 | 340 | 20 | 1860 | 1860

The simulations make me even more confused:
1. The blood/plasma ratio varies little for different methods, but all the cases show an unreasonable high blood/plasma ratio when LogP=3.5. For a compound having lipophilicity >3.5 or 3, most of the drug distributes into blood cells rather than tissues. Plasma PK will be meaningless.
2. Clearance varies dramatically for different methods. This makes no sense because clearance should be independent of the volume of distribution.

**Comment by tobiasK2001** [comment]
Dear jzq90,

yes this may be confusing on the first glance. And you rise an interesting point here:
> Clearance varies dramatically for different methods. This makes no sense because clearance should be independent of the volume of distribution.
But if you think that this concept"clearance should be independence from Vss" comes from "top down" compartimental and non compartimental PK-analysis. This uses some inherent assumptions like: 

- you look on plasma only  
- drug binding in nonvascular regions (i.e. tissue binding) does not play a role.
- the rest of the body consists 1, 2 or three well stirred compartments
These assumptions do not hold for PBPK modelling were you have a "bottom up approach". Here compound parameters like lipophilicity impact drug fate in the body, which is described by e.g. drug permeability from plasma to tissues with different velocities and tissue binding. PK-Sim calculates the PK parameters directly from the plasma curve were all the dynamics are reflected. If you realize that the different tissue distribution models use slightly different approaches to predict this behavior, it makes sense, that they lead to different results.

Hope this might help you.

Best, Tobias

**Comment by jzq90** [comment]
Dear Tobias,
Thanks for your kind reply. But I have too many questions about this:

>     These assumptions do not hold for PBPK modelling were you have a "bottom up approach". Here compound parameters like lipophilicity impact drug fate in the body, which is described by e.g. drug permeability from plasma to tissues with different velocities and tissue binding. PK-Sim calculates the PK parameters directly from the plasma curve were all the dynamics are reflected. If you realize that the different tissue distribution models use slightly different approaches to predict this behavior, it makes sense, that they lead to different results.

1. Does the partition method affect the permeability？
2. Still, I can't understand why the partition to the tissue influence the clearance (e.g. AUC in the plasma). Elimination of the drug only occurs in the liver. If the drug partition more to the tissue, the free drug in plasma will decrease, then the elimination rate (CL×C) will decrease. All the drug partition into the tissue will flow back to plasma, which should keep the AUC unchanged.
3. The Vss varies only within around 5-fold range, while the clearance varies up to 400-fold for different partition method. Why?
4. The blood/plasma ratio is unreasonable high. 

Best regards,
Aaron

**Comment by StephanSchaller** [comment]
Hi Aaron,

1. No
2. No, if the clearance is in liver , the concentration in plasma is not affecting clearance, but the concentration in liver intracellular, which may be changed with changes in partitioning (e.g the drug reaches the liver faster).
3 & 4: can you share the project file?

**Comment by jzq90** [comment]
Hi StephanSchaller

> 
>  2. No, if the clearance is in liver , the concentration in plasma is not affecting clearance, but the concentration in liver intracellular, which may be changed with changes in partitioning (e.g the drug reaches the liver faster).

If the partition method doesn't affect permeability, then they will reach the liver at the same speed (**not faster**). Isn't it?

I've uploaded the project as below:
https://github.com/jzq90/Partition-Model

As described above, this is a dummy compound with Log P=3.5 and low intrinsic clearance. The tests were compared among different **partition methods**.

  | PK-Sim standard | Rodgers and Rowland | Schemitt | Poulin and Theil | Berezhkovskiy
-- | -- | -- | -- | -- | --
Plasma   clearance (mL/min/kg) | 78.4 | 3.61 | 93.14 | 0.20 | 0.20
Blood   clearance  (mL/min/kg) | 4.26 | 0.2 | 3.66 | 0.01 | 0.01
Blood/Plasma | 18.4 | 18.1 | 25.4 | 20.0 | 20.0
Plasma   Vss (L/kg) | 236 | 106 | 125 | 31.6 | 31.6
Blood   Vss  (mL/min/kg) | 9.83 | 5.75 | 4.93 | 1.72 | 1.72
Thalf   (h) | 35 | 340 | 20 | 1860 | 1860

**Comment by StephanSchaller** [comment]
> If the partition method doesn't affect permeability, then they will reach the liver at the same speed (not faster). Isn't it?

No, partitioning also affects diffusion into cell. it changes the gradient of the diffusive flow, i.e. speed of diffusion.

**Comment by jzq90** [comment]
> 
> No, partitioning also affects diffusion into cell. it changes the gradient of the diffusive flow, i.e. speed of diffusion.

I thought the partition only affects the steady-state, in which the Kp = Drug-in-tissue/Drug-in-blood keeps unchanged. But before it reach the steady-state, the permeation rate and gradient should be the same for different partition method.

**Comment by prvmalik** [comment]
The rate of diffusion from compartment 1 to compartment 2 is PSA x (C1 - C2/K21)

As K21 increases, the rate of diffusion (permeation rate) increases, and vice versa.

**Comment by StephanSchaller** [comment]
Thanks Paul. Forgot to follow-up, here

**Comment by jzq90** [comment]
> PSA x (C1 - C2/K21)

Hi, thanks. What's the meaning of K21 and the equation? I thought the drug flowing from compartment 1 to compartment 2 should be PSA x C1 with unit ng/s and the drug flowing from compartment 2 to compartment 1 should be PSA x C2 with unit ng/s.

Also, why K21 increases when change the partition method (not permeability method) ?

**Comment by prvmalik** [comment]
Getting off track but...

K21 is the partition coefficient between compartment 2 and compartment 1. In other words, it is the ratio of the drug concentration in compartment 2 vs. compartment 1 at a distributional steady state. Its determination is dependent on the partition coefficient algorithm that you select (e.g. Rodgers and Rowland, PK-Sim Standard and so on).

You are almost correct about the flow, but you must account for the partition coefficient. If you expand the differential equation for C2 you get:

d/dt(C2) = PSA x (C1 - C2/K21)
d/dt(C2) = PSA x C1 - PSA x C2/K21

And the opposite for C1:

d/dt(C1) = -PSA x (C1 - C2/K21)
d/dt(C1) = -PSA x C1 + PSA x C2/K21

**Comment by jzq90** [comment]
> at a distributional steady state
Yes, if you mean steady-state it makes sense, and definitely the partition method influences the tissue/plasma ratio. But like I mentioned, 'I thought the partition only affects the steady-state... But before it reaches the steady-state, the permeation rate and gradient should be the same for different partition method'

Also, K21= C2/C1, then  (C1 - C2/K21) will be 0.

**Comment by StephanSchaller** [comment]
Yes, change (diffusive flow) will be zero if the drug is in steady state.

And the calculated permeability is part of the "PSA" = "permeability x surface area factor".

So PSA defines the speed and K21 defines the ratio (and one-directional speed).

**Comment by jzq90** [comment]
OK. Back to my original question:

Why does the partition method affect the clearance (both blood and plasma) to such a huge extent (400-fold), even larger than the change-fold of volume distribution?

**Comment by dzianismr** [comment]
@jzq90 
I think this is a numerical aritifact caused by the unrealistic physchem properties of the dummy compound (highly lipophilic compounds tupically show substantial protein binding, wheres fu for the dummy was set to 1). In this case calculated volumes of disitrubution exceed 100L and half lifes are also extreme. If you would like to have accurate CL estimate based on NCA you will need to have a very long  observation period.
